packages/iov-lisk/src/liskcodec.ts

```diff
@@ -31,8 +31,8 @@ export const liskCodec: TxCodec = {
    * https://github.com/prolina-foundation/snapshot-validator/blob/35621c7/src/transaction.cpp#L36
    */
   bytesToSign: (unsigned: UnsignedTransaction, nonce: Nonce): SigningJob => {
-    const creationTimestamp = nonce.toNumber();
-    const creationDate = new ReadonlyDate(creationTimestamp * 1000);
+    const creationTimestamp = new Int53(nonce);
+    const creationDate = new ReadonlyDate(creationTimestamp.toNumber() * 1000);
```

Comment:
Can we leave the safe range here (and below)?

Reply:
Yes, we can, good point. Unfortunatly as far as I know, we cannot construct a JS Date using seconds directly.

But this can only happen for extreme times that are not in the Lisk time range (Lisk epoch plus int32)